### DISCUSSION TITLE
<title> Arquivos não estão funcionando  </title>

### DISCUSSION THREAD

**Discussion by osmaclean** [community/community#50018] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Olá bom dia, gostaria de pedir uma ajuda à comunidade, realizei um projeto final de semana e subi os arquivos para o git, entretanto, o link não abre na aba "Pages", fiz o deploy no vercel e também está quebrado, alguém pode me ajudar? Não entendi o que ocorreu... o link do repositório → https://github.com/osmaclean/adegaDoPericles_DNC

**Comment by neutrinomuon** [comment, marked as answer]
Olá... Verifiquei o deploy e aparentemente está tudo funcionando corretamente. Porém, é importante ressaltar que às vezes pode demorar um pouco para as atualizações serem aplicadas. Além disso, se você realizar muitos uploads seguidos de arquivos para o Git, pode ocorrer erros e a atualização pode demorar um pouco mais do que o normal.
